### PR TITLE
Frame level

### PR DESCRIPTION
**Pull request by enajork**
Modules/Frames should now properly obscure each other (especially their contents) depending on which one has focus.

I did not test this thoroughly but I believe this is the correct solution to the problem.

### CODE REVIEW

**Comment by enajork** [comment]
It would also be great to get this fix into the Classic version as well. I see that the Classic version uses 7968cf257ef6cf4619367a575e5251649dfcda63 and might not benefit from these changes immediately. Thank you.

**Comment by enajork** [comment]
Whoops closed by accident. Sorry for spam.

**Comment by enajork** [comment]
I tested the session, voting, and history frames on Retail and Classic. Everything looks good. Some other modules/frames may still need to be tested.

File: Modules/sessionFrame.lua [hunk 1]
```diff
@@ -252,7 +252,7 @@ function RCSessionFrame:GetFrame()
 	f.lootStatus.text:SetJustifyH("LEFT")
 
 	local st = ST:CreateST(self.scrollCols, 5, ROW_HEIGHT, nil, f.content)
-	st.frame:SetPoint("TOPLEFT",f,"TOPLEFT",10,-20)
+	st.frame:SetPoint("TOPLEFT",f,"TOPLEFT",10,-ROW_HEIGHT-10)
```
Comment (enajork) [inline Modules/sessionFrame.lua:255]: Could just make this -50. No particular reason I did it this way.

**Comment by evil-morfar** [comment]
I've been wanting to look into this for quite a while, so thanks a lot!

Classic is still based on the v2 design, which doesn't have the frames in the same place. Shouldn't be too hard to implement there, as all I more or less copy/pasted the code straight into `Widgets/Frame.lua`.

**Comment by enajork** [comment]
I implemented the fix in v2 first then in v3. Just didn't want to bother creating a separate pull request for v2 (seemed more complicated). Figured you would copy pasta for the Classic version 😄 

Thank you so much!

**Pull request merged**